pkg/packages.proj

```diff
@@ -6,12 +6,23 @@
     <BuildAllOSGroups Condition="'$(FilterToOSGroup)' != ''">false</BuildAllOSGroups>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(SkipManagedPackageBuild)' != 'true'">
+  <PropertyGroup Condition="'$(OfficialBuildId)' != ''">
+    <!-- During an official build, only build identity and targeting packages in the Windows build -->  
+    <BuildIdentityPackage Condition="'$(BuildIdentityPackage)' == '' AND '$(OS)' != 'Windows_NT'">false</BuildIdentityPackage>
+    <BuildIdentityPackage Condition="'$(BuildIdentityPackage)' == '' AND '$(OS)' == 'Windows_NT' AND '$(BuildArch)' == 'x64'">true</BuildIdentityPackage>
+  </PropertyGroup>
+
+  <!-- Targeting pack to be consumed by CoreFX - intentionally produce only a Windows x64 version -->
```

Comment:
Is this what CoreCLR does for the targeting pack?

Reply:
Yes. If doing and official build the BuildIdentityPackage property is set , and every package with an unset PackageTargetRuntime is built only under Windows_NT x64. The difference there is that it's a generic process, which filters packages to be built - ideally we'd port this to CoreRT at some point.

Reply:
If you meant the comment - we do this anyway to avoid different APIs in system libraries across platforms and architectures. The targeting pack was being built for Mac and Linux even though its wasn't being consumed.